src/org/usfirst/frc4904/robot/subsystems/GearIO.java

```diff
@@ -0,0 +1,37 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.Servo;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class GearIO extends Subsystem {
+	public Servo gearFlap;
+	public DoubleSolenoid gearSlotOpener;
+	
+	public GearIO(Servo gearFlap, DoubleSolenoid gearSlotOpener) {
+		this.gearFlap = gearFlap;
+		this.gearSlotOpener = gearSlotOpener;
+	}
+	
+	public void IntakeGear() {
+		gearSlotOpener.set(DoubleSolenoid.Value.kForward);// closes gear stuff
+		gearFlap.setAngle(90);// might be inaccurate--will test
+	}
+	
+	public void OuttakeGear() {
+		gearFlap.setAngle(0);// closes gearFlap
+		gearSlotOpener.set(DoubleSolenoid.Value.kReverse);// opens gear holder
+	}
+	// Put methods for controlling this subsystem
+	// here. Call these from Commands.
+	
+	@Override
+	public void initDefaultCommand() {
+		// Set the default command for a subsystem here.
```

Comment:
Add `setDefaultCommand(new Idle());` here. This means that when any running command on the subsystem is cancelled, the Idle command runs instead. Idle literally does nothing: if a motor is running, it won't change it.